### ISSUE TITLE
[NBug] An error occurred trying to start process....

### ISSUE THREAD

**Issue by RussKie** [gitextensions/gitextensions#10728]
### Environment

- Git Extensions 4.0.1.15887
- Build f2567dea28bbf0e2cbcd2a95d4c7200e47d0d671
- Git 2.39.1.windows.1
- Microsoft Windows NT 10.0.22621.0
- .NET 6.0.13
- DPI 96dpi (no scaling)

### Issue description

While trying to open the link to the AzDO PATs I get this:
![image](https://user-images.githubusercontent.com/4403806/218881716-4d1164ea-6718-40b3-b6d7-d8af7ae4aedd.png)

```
System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'https://xyz.visualstudio.com/_details/security/tokens' with working directory 'C:\WINDOWS'. The system cannot find the file specified.
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   at System.Diagnostics.Process.Start(String fileName)
   at AzureDevOpsIntegration.Settings.SettingsUserControl.RestApiTokenLink_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Label.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

### Steps to reproduce

Settings > Build server integration > Build server type = Azure DevOps > Got to token management page

**Comment by RussKie** [comment]
This is similar to https://github.com/gitextensions/gitextensions/issues/10726

**Comment by RussKie** [comment]
Fixed in #10727
